From 01f48cca4dfe9f980c34070ade0988d5b7614ed0 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 12:35:41 +0800
Subject: [PATCH] Test case6 for Change Pcd value in PcdsFeatureFlag

Description: Change Pcd value in PcdsFeatureFlag section (to gEfiMdeModulePkgTokenSpaceGuid.PcdSupportUpdateCapsuleReset|FALSE|BOOLEAN|0x0001001e) in MdeModulePkg/MdeModulePkg.dec.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/MdeModulePkg.dec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdeModulePkg/MdeModulePkg.dec b/MdeModulePkg/MdeModulePkg.dec
index 843e963ad3..4731c29ef8 100644
--- a/MdeModulePkg/MdeModulePkg.dec
+++ b/MdeModulePkg/MdeModulePkg.dec
@@ -629,7 +629,7 @@
   #   TRUE  - Supports update capsule across a system reset.<BR>
   #   FALSE - Does not support update capsule across a system reset.<BR>
   # @Prompt Enable update capsule across a system reset.
-  gEfiMdeModulePkgTokenSpaceGuid.PcdSupportUpdateCapsuleReset|FALSE|BOOLEAN|0x0001001d
+  gEfiMdeModulePkgTokenSpaceGuid.PcdSupportUpdateCapsuleReset|FALSE|BOOLEAN|0x0001001e
 
   ## Indicates if all PCD PPI services will be enabled.<BR><BR>
   #   TRUE  - All PCD PPI services will be produced.<BR>
-- 
2.27.0.windows.1

